pkg/server/commands/list_objects.go

```diff
@@ -36,6 +36,11 @@ type ListObjectsQuery struct {
 	ResolveNodeLimit      uint32
 }
 
+type ObjectOrError struct {
```

Comment:
Can we please rename to `ListObjectsResult`?

Reply:
Resolved in https://github.com/openfga/openfga/pull/779/commits/179f7dd2f538126a85d7cdfc59ee06bba53f0590